library/build.gradle

```diff
@@ -55,7 +55,14 @@ dependencies {
     // https://mvnrepository.com/artifact/io.reactivex.rxjava2/rxkotlin
     implementation("io.reactivex.rxjava2:rxkotlin:2.4.0")
 
-    testImplementation group: 'junit', name: 'junit', version: '4.12'
+    // https://mvnrepository.com/artifact/jakarta.xml.bind/jakarta.xml.bind-api
+    implementation 'jakarta.xml.bind:jakarta.xml.bind-api:2.3.3'
```

Comment:
Hmmm how will it end up for Java 8 users tho? 🤔

Reply:
I'm not sure how it's gonna interact with the built-in `javax.xml.bind` classes 💥